qutebrowser/completion/completionwidget.py

```diff
@@ -192,10 +192,13 @@ def _next_prev_item(self, prev):
         Args:
             prev: True for prev item, False for next one.
         """
-        idx = self._next_idx(prev)
-        qtutils.ensure_valid(idx)
-        self.selectionModel().setCurrentIndex(
-            idx, QItemSelectionModel.ClearAndSelect | QItemSelectionModel.Rows)
+        # selmodel can be None if 'show' and 'auto-open' are set to False
+        # https://github.com/The-Compiler/qutebrowser/issues/1731
+        selmodel = self.selectionModel()
+        if (selmodel is not None):
+            idx = self._next_idx(prev)
```

Comment:
I guess we should return here if the index is invalid, as otherwise we're calling `setCurrentIndex` with an invalid index?

Something like:

```python
if not idx.isValid():
    return
```